cabal-testsuite/PackageTests/ProjectConfig/FieldStanzaConfusion/cabal.test.hs

```diff
@@ -0,0 +1,6 @@
+import Test.Cabal.Prelude
+
+main = cabalTest $ do
+  result <- fails $ cabal' "build" []
+  assertOutputContains "Error parsing project file" result
```

Comment:
I wonder if these should start using error numbers. (That might be a separate PR, though.)

Reply:
It's error 7090 (`ReportParseResult`). Not super specific but there is a number for it.

Reply:
Oh. Perhaps future work could be making parse errors more specific; one error number for all of them seems like it would make `errors.haskell.org` less useful.